src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -51,6 +51,9 @@
 
     private final String replacementStrategy;
     private final boolean taggable;
+    private final ResourceTagging tagging = new ResourceTagging(true, true, true,
```

Comment:
Could we have a default ResourceTagging, so we can do `ResourceTagging.DEFAULT`

Reply:
sure, let me add it